RENotebookApi/RENotebookApi.fsproj

```diff
@@ -20,6 +27,12 @@
     <None Include="paket.references" />
     <None Include="ReLoad.fsx" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="AutomaticGraphLayout" Version="1.1.9" />
```

Comment:
We should probably keep to either nuget direct references or Paket. Combining leads to madness